sos/collector/clusters/saltstack.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Red Hat 2022, Trevor Benson <trevor.benson@gmail.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import json
+from shlex import quote
+from sos.collector.clusters import Cluster
+
+
+class saltstack(Cluster):
+    """
+    The saltstack cluster profile is intended to be used on saltstack
+    clusters (Salt Project).
+    """
+
+    cluster_name = "Saltstack"
+    packages = ("salt-master",)
+    sos_plugins = ["saltmaster"]
+    cmd = "salt-run --out=pprint manage.status"
```

Comment:
Two suggestions, leaving on author to decide:

- move this variable assignment below `option_list`; i.e. declare variables common to all `Cluster` classes first, then declare the `Saltstack` specific ones
- rename the option to e.g. `node_cmd` or `node_list_cmd`; `cmd` is too generic and bit confusing with (non-class instance) `cmd` variable `cmd = fqdn_cmd.format(minion=minion)`.